### ISSUE TITLE
[Next] Lutris doesn't launch when steam sync is enabled

### ISSUE THREAD

**Issue by Medath** [lutris/lutris#1366]
Steps to reproduce:
Have steam sync, winesteam sync, or desktop games sync enabled and launch lutris next (265335d)

Expected behaviour:
Lutris launches.

Actual behaviour:
It gives the following output and quits (In this example desktop game sync was enabled):
```
matze@solus-ob ~/Schreibtisch/git-projects/lutris/real $ bin/lutris -d
2018-11-23 14:21:31,272: Running Lutris 0.5.0-alpha
Traceback (most recent call last):
  File "/home/matze/Schreibtisch/git-projects/lutris/real/lutris/gui/application.py", line 247, in do_activate
    self.window = LutrisWindow(application=self)
  File "/home/matze/Schreibtisch/git-projects/lutris/real/lutris/gui/lutriswindow.py", line 121, in __init__
    service.sync_with_lutris()
TypeError: sync_with_lutris() missing 1 required positional argument: 'games'
Traceback (most recent call last):
  File "/home/matze/Schreibtisch/git-projects/lutris/real/lutris/gui/application.py", line 384, in do_command_line
    self.window.present()
AttributeError: 'NoneType' object has no attribute 'present'
INFO     2018-11-23 14:21:31,399 [application.do_shutdown:475]:Shutting down Lutris
```
Additional information:
Reverting back to d4049ea makes Lutris launch again, so the bug must be in 5c5c40c

**Comment by Medath** [comment]
Crashing got fixed by #1376. However, importing games still does not work.

**Comment by strycore** [comment]
Not a bug, I'm currently working on it.
